modules/kmm-replacing-in-tree-modules-with-out-of-tree-modules.adoc

```diff
@@ -10,7 +10,8 @@ You can use Kernel Module Management (KMM) to build kernel modules that can be l
 
 Dynamically loaded modules include in-tree modules and out-of-tree (OOT) modules. In-tree modules are internal to the Linux kernel tree, that is, they are already part of the kernel. Out-of-tree modules are external to the Linux kernel tree. They are generally written for development and testing purposes, such as testing the new version of a kernel module that is shipped in-tree, or to deal with incompatibilities.
 
-Some modules loaded by KMM could replace in-tree modules already loaded on the node. To unload an in-tree module before loading your module, set the `.spec.moduleLoader.container.inTreeModuleToRemove` field. The following is an example for module replacement for all kernel mappings:
+Some modules loaded by KMM could replace in-tree modules already loaded on the node. To unload in-tree modules before loading your module, set `.spec.moduleLoader.container.inTreeModulesToRemove`.
```

Comment:
```suggestion
Some modules that are loaded by KMM could replace in-tree modules that are already loaded on the node. To unload in-tree modules before loading your module, set `.spec.moduleLoader.container.inTreeModulesToRemove`.
```

`.spec.moduleLoader.container.inTreeModulesToRemove` should be treated as an adjective, e.g. "set the value of the `.spec.moduleLoader.container.inTreeModulesToRemove` key to <value>`"

Reply:
Changed to "To unload in-tree modules before loading your module, set the value of `.spec.moduleLoader.container.inTreeModulesToRemove` accordingly."